lib/services/powerbiembedded/generate.cmd

```diff
@@ -0,0 +1,19 @@
+::
+:: Microsoft Azure SDK for Node - Generate library code
```

Comment:
We don't use generate.cmd, instead we use a gulp file. There is an entry for powerbiembedded over [here](https://github.com/Azure/azure-sdk-for-node/blob/master/gulpfile.js#L118). Please use the steps mentioned [here]( https://github.com/Azure/azure-sdk-for-node/blob/master/Documentation/RegeneratingCode.md) to regenerate the sdk. Before regenerating, please pull from upstream master.